static-assets/scripts/admin.js

```diff
@@ -601,7 +616,23 @@
 
       CrafterCMSNext.system.getStore().subscribe((store) => {
         $scope.sites = store.getState().sites.byId;
-        audit.getAuditInfo();
+
+        // TODO: we need store in order to get activeSite, when updating to a global config, this can be a forkJoin
+        const activeSite = store.getState().sites.active;
+        sitesService.fetchSiteLocale(activeSite).then((locale) => {
+          if (Object.keys(locale).length === 0) {
+            $scope.locale = {
+              localeCode: 'en-US',
+              dateFormatOptions: {
+                timeZone: 'EST5EDT'
+              }
```

Comment:
This wouldn't show any time info. We need some additional defaults to make sure time too is shown.
Call me to discuss.